jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/ChipsTests.java

```diff
@@ -115,22 +143,49 @@ public void customListChipTests() {
     public void expandableChipTests() {
         assertThat(expandableMenu.isNotDisplayed(), Matchers.is(true));
         expandableChip.has().image();
-
         expandableChip.click();
         waitCondition(() -> expandableMenu.isDisplayed());
         assertThat(expandableMenu.isDisplayed(), Matchers.is(true));
         expandableMenuItems.get(1).click(); // collapse menu just to be safe
     }
 
     @Test
-    public void inSelectChipTests() {
+    public void labelTextChipTests() {
```

Comment:
This test for custom realization -> I suppose we don't need it

Reply:
As Natalia suggested, I moved custom realization to custom folder in tests module